.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: node_js
 node_js:
   - 0.10
+befofe_script:
```

Comment:
Yeah, I was looking at that. But I also read that this change alone should be sufficient for Firefox, which doesn't seem to work.